emcc.py

```diff
@@ -2007,6 +2007,9 @@ def check_human_readable_list(items):
       # requires JS legalization
       shared.Settings.LEGALIZE_JS_FFI = 0
 
+    if shared.Settings.WASM_BIGINT:
+      shared.Settings.LEGALIZE_JS_FFI = 0
```

Comment:
Is legalization *only* dealing with the i64 issue?

Reply:
We used to legalize f32s to doubles for JS too, but it turned out that was not necessary. So it is just i64s now, yes.